src/util/scenegraph/parse.ts

```diff
@@ -222,12 +222,6 @@ export function parse(
   decimalPlaces: number,
   annotations: Annotation[],
 ): Scenegraph | null {
-  if (isVConcatSpec(specification) && specification.vconcat.length > 2) {
```

Comment:
@johnwalley we'd like to set two studies by default, but I couldn't see a reason why this would be bad idea but this warning indicates otherwise.. could you shed some light on it?

Reply:
Hmmm. The warning looks wrong. Perhaps it dates from when we only had one study. You're right to remove it.